2377/2/versus/battles_class.rb

```diff
@@ -0,0 +1,35 @@
+# This class keeps the data of one battle
+class Battle
+  attr_reader :words_per_round, :filename
+  def initialize(filename)
+    @filename = filename
+    @line_array = make_line_array
```

Comment:
the typical pattern for things like this is to create `line_array` method that will return what you need. To avoid performance issues read about memoization in ruby (`||=`)